materialchart/src/main/java/app/priceguard/materialchart/Chart.kt

```diff
@@ -517,6 +516,7 @@ class Chart @JvmOverloads constructor(
         chartData.forEachIndexed { index, data ->
             if (index < size - 1) {
                 val next = chartData[index + 1]
+                linesPaint.color = if(data.valid) colorPrimary else colorError
```

Comment:
```suggestion
                linesPaint.color = if(data.valid) {
                    colorPrimary
                } else {
                    colorError
                }
```

Please change this to proper if format